src/l10n/cs.js

```diff
@@ -0,0 +1,338 @@
+(function() {
+    let translations = {
+            'Passwords'                           : 'Správce hesel',
+            'All'                                 : 'Vše',
+            'Folders'                             : 'Složky',
+            'Recent'                              : 'Nedávné',
+            'Favourites'                          : 'Oblíbené',
+            'Shared'                              : 'Sdílené',
+            'Tags'                                : 'Značky',
+            'Security'                            : 'Bezpečnost',
+            'Trash'                               : 'Koš',
+            'Details'                             : 'Podrobnosti',
+            'Edit'                                : 'Upravit',
+            'Rename'                              : 'Přejmenovat',
+            'Delete'                              : 'Smazat',
+            'Restore'                             : 'Obnovit',
+            'New Password'                        : 'Nové heslo',
+            'New Folder'                          : 'Nová složka',
+            'New Tag'                             : 'Nová značka',
+            'Secure'                              : 'Bezpečné',
+            'Weak'                                : 'Slabé',
+            'Broken'                              : 'Velmi slabé',
```

Comment:
If i'm correct, "Velmi slabé" translates to "Very weak". Is there a stronger alternative for this? The idea is to tell the user that passwords in this category are so bad, you can't consider them to keep anyone from breaking them.

Reply:
Yes, you are correct "Velmi slabé" translates as "Very weak". "Broken" could be translated as "Rozbité" without context - but that would not make sense here. I think that in this situation "Broken" should be translated as "Nebezpečné" (dangerous) or "Velmi nebezpečné" (very dangerous). Should I change this in my repository and create new pull request? I'm new to github, so I'm sorry for this question.

Reply:
Great. You can just change it in your repository and github will update the pull request as soon as you push the changes.

Reply:
OK, done.